templates/skilltrees.html

```diff
@@ -0,0 +1,46 @@
+{%extends "layout.html"%}
+	{%block head%}
+		<title>CX Docs - Skill Trees</title>
+	{% endblock %}
+	{% block body %}
```

Comment:
Code Police have asked me to relay to you that you left spaces out of your head tag.